lib/shipit/command.rb

```diff
@@ -215,5 +214,40 @@ def parse_arguments(arguments)
       end
       return args, options
     end
+
+    def running?
+      !!pid && !@status
+    end
+
+    def code
+      @status&.exitstatus
```

Comment:
The `&` guard is to maintain the same API that the previous `attr_reader` provided.